pkg/gui/recording.go

```diff
@@ -21,7 +21,7 @@ func recordEventsTo() string {
 }
 
 func (gui *Gui) timeSinceStart() int64 {
-	return time.Since(gui.StartTime).Milliseconds()
+	return time.Since(gui.StartTime).Nanoseconds() / 1e6
```

Comment:
shouldn't this be `* 1e6`

Reply:
I got this from here https://golang.org/src/time/time.go?s=24262:24300#L725

Also nano is less than mili

Reply:
goes to show what happens when you haven't touched maths since university